src/_index.js

```diff
@@ -356,13 +356,35 @@ class EasySelect{
         if(this.isDisabled) return false;
 
         // avoid duplicate value
-        if(this.selectTagData.filter(option => option.value === value).length > 0){
-            if(this.config.warning) console.warn(`[ES] ${value} will not be added due to duplicating`);
-            return false;
+        const isDuplicate = (value) => {
+            if(this.selectTagData.filter(option => option.value === value).length > 0){
+                if(this.config.warning) console.warn(`[ES] ${value} will not be added due to duplicating`);
+                return true;
+            }
         }
 
         // add new option to select tag
-        this.selectTag.insertAdjacentHTML('beforeend', `<option value="${value}">${value}</option>`);
+        const addNewOption = (value, label) => {
+            if(typeof value === 'object'){
+                return this.selectTag.insertAdjacentHTML('beforeend', `<option value='${value}'>${label}</option>`);
```

Comment:
What is the return value of this function? The old version did not return anything.